### PR TITLE
Future forecast changes report

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Domain.LinnApps/Reports/ForecastWeekChangesReportService.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                "Percentage Changes applied to the coming weeks: ");
+
+            reportLayout.AddColumnComponent(
+                null,
+                new List<AxisDetailsModel>
+                    {
+                        new AxisDetailsModel("EndsOn", "Week Ending",  GridDisplayType.TextValue),
+                        new AxisDetailsModel("PercentageChange", "% Change",  GridDisplayType.Value)
+                            {
+                                DecimalPlaces = 2
```
Comment (richardiphillips) [inline src/Domain.LinnApps/Reports/ForecastWeekChangesReportService.cs:43]: could add a % suffix

File: src/Domain.LinnApps/LinnWeek.cs [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
+
     public class LinnWeek
     {
         public int WeekNumber { get; set; }
 
         public DateTime StartsOn { get; set; }
 
         public DateTime EndsOn { get; set; }
+
+        public ForecastWeekChange ForecastChange { get; set; }
```
Comment (richardiphillips) [inline src/Domain.LinnApps/LinnWeek.cs:15]: this can't be a one to one relation can it?

**Review by richardiphillips — APPROVED** [review]

File: src/Domain.LinnApps/LinnWeek.cs (hunk 2 above)
Comment (lewisrenfrew) [reply src/Domain.LinnApps/LinnWeek.cs:15]: I had assumed it was! One % change per week I thought? Or can a week have multiple different values?

Comment (richardiphillips) [reply src/Domain.LinnApps/LinnWeek.cs:15]: oh yeah. you're completely correct of course.

**Pull request merged**